test/integration/ovs/ofctrl_test.go

```diff
@@ -355,6 +355,51 @@ func TestBundleErrorWhenOVSRestart(t *testing.T) {
 	require.Equal(t, 0, failCount, "No fail case expected")
 }
 
+func TestReconnectOFSwitch(t *testing.T) {
```

Comment:
suggested comment:

TestReconnectOFSwitch verifies that the OpenFlow connection to OVS can be restored, even when OVS is down for a long amount of time.

Reply:
Thanks for giving suggesions for the comment, will add it.